packages/react/src/lib/accessibility/FocusZone/FocusTrapZone.tsx

```diff
@@ -296,13 +296,19 @@ export class FocusTrapZone extends React.Component<FocusTrapZoneProps, {}> {
       )
     }
 
-    // loop through all body's children, except the last one - which is the popup
-    for (let index = 0; index < bodyChildren.length - 1; index++) {
-      const element = bodyChildren[index]
-
-      if (element.getAttribute('aria-hidden') !== 'true') {
-        element.setAttribute('aria-hidden', 'true')
-        element.setAttribute(HIDDEN_FROM_ACC_TREE, 'true')
+    for (let index = 0; index < bodyChildren.length; index++) {
```

Comment:
does a for ... of loop work? also with type included. something like:

```suggestion
    for (const currentChild of bodyChildren as HTMLElement[]) {
```

Reply:
for..loop works and it's the fastest method of looping  (if for e.g. to compare to forEach). But I like your way too, will change it. Thank you!

Reply:
If you do use for/of, please test with IE, see https://stackoverflow.com/questions/22754315/for-loop-for-htmlcollection-elements

Reply:
We use Babel transpiler of ES6, aren't we?

Reply:
@layershifter is working on that. In meantime, will keep basic loop